etna/datasets/internal_datasets.py

```diff
@@ -168,14 +168,15 @@ def load_dataset(
                 "copy of the dataset reflects a more recent version of the data than your version of the library. "
                 "In this case you can try updating the library version."
             )
-        if _check_dataset_local(dataset_dir / EXOG_SUBDIRECTORY):
+        if _check_dataset_local(dataset_dir / EXOG_SUBDIRECTORY / f"{name}_{part}_exog.csv.gz"):
             df_exog = pd.read_csv(
                 dataset_dir / EXOG_SUBDIRECTORY / f"{name}_{part}_exog.csv.gz",
                 compression="gzip",
                 header=[0, 1],
                 index_col=[0],
                 parse_dates=[0],
             )
+            df_exog = df_exog.astype("datetime64[ns]")
```

Comment:
Are we sure that the only thing we will have in `df_exog` in the future will be timestamp?

Reply:
We can rewrite this in the future, if it won't be true.